tests/test_workflow.py

```diff
@@ -708,6 +708,7 @@ def test_get_jobs_live(workflow_live):
     jobcollection = workflow_live.get_jobs()
     assert isinstance(jobcollection, list)
     assert isinstance(jobcollection.jobs[0], Job)
+    # pylint: disable=use-a-generator
     assert all(
         [j._info["workflowId"] == workflow_live.workflow_id for j in jobcollection.jobs]
     )
```

Comment:
You can avoid this disable by removing the list inside all. i.e.
`assert all(j._info["workflowId"] == workflow_live.workflow_id for j in jobcollection.jobs)`

Reply:
thanks! Done!